packages/ui/src/theme/tokens/components/tabs.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
```

Comment:
All of these styles are my best guesses. Definitely needs review from @dbanksdesign.

Reply:
I went through and updated the styles, only a few minor tweaks but overall great job @joebuono !